TombEngine/Objects/TR5/Entity/tr5_hydra.cpp

```diff
@@ -324,7 +325,7 @@ namespace TEN::Entities::Creatures::TR5
 
 				if (!(GlobalCounter & 3))
 				{
-					frame = ((GetAnimData(item).frameBase - item->Animation.FrameNumber) / 8) + 1;
+					frame = (item->Animation.FrameNumber / 8) + 1;
```

Comment:
I believe this won't work. Original code (https://github.com/Trxyebeep/TOMB5/blob/0aeec3e255a53b48198f571c02995e0471c564cf/TOMB5/game/hydra.cpp#L197) used a bitshift on a negative value to get specific effect pattern. We don't use bitshifts, and we don't have negative values anymore. So the effect needs to be checked.